metsim/cli/ms.py

```diff
@@ -63,7 +63,14 @@ def invert_dict(d):
     def to_list(s):
         return json.loads(s.replace("'", '"').split('#')[0])
 
+
+    conf['constant_vars'] = OrderedDict(config['constant_vars'])
     conf['forcing_vars'] = OrderedDict(config['forcing_vars'])
+    # remove constant_vars from forcings_vars
+    del_keys = list(set(conf['forcing_vars'].keys()).
+                    intersection(conf['constant_vars'].keys()))
+    [conf['forcing_vars'].pop(x) for x in del_keys]
```

Comment:
This way the `constant_vars` are never even read even if they are also mentioned in `forcing_vars`

Reply:
Actually that does not work with the binary and ascii reads in which case you *must* read all the met variables. So nice but deleted in later update